CMakeLists.txt

```
@@ -59,6 +58,10 @@ elseif(CMAKE_BUILD_TYPE STREQUAL "Debug")
     )
 endif()
 
+if(packaging)
+	set(build-number "0" CACHE STRING "Build number incremented from CI counter at the fourth position in the version number, stable = 0.")
```

"stable = 0" -> That's not what we concluded yesterday. According to our discussion yesterday, the stable release does not have any particular build number. It will simply end up having the build number that is up next in the row.
Besides: We should not describe cmake variables by saying that this is something "the CI uses".

Furthermore given that 0 is a valid (and arbitrary) build number, we should probably default the build number to an empty String, so that we can warn/error if the user specified `packaging` but did not specify a build number :point_up: 

```suggestion
	set(build-number "" CACHE STRING "The build number of the current build. Will be used in Mumble's version to make sure newer builds upgrade older installations properly.")
	
	if (build-number STREQUAL "")
		message(FATAL_ERROR "Tried to create a Mumble package, without specifying build-number!")
	endif()
```
In addition we should extend the `packaging` description to state that the build number has to be specified when using this option.
Alternatively we could make this a warning and then default to a build number of 0 in this case.